src/widget/wsearchlineedit.cpp

```diff
@@ -233,7 +225,7 @@ void WSearchLineEdit::loadQueriesFromConfig() {
             m_pConfig->getKeysWithGroup(kSavedQueriesConfigGroup);
     QSet<QString> queryStrings;
     for (const auto& queryKey : queryKeys) {
-        const auto& queryString = m_pConfig->getValueString(queryKey);
+        const auto& queryString = m_pConfig->getValueString(queryKey).simplified();
```

Comment:
A query may intentionally contain multiple spaces and they might be quoted. They are not redundant and we should not remove them. Only leading/trailing spaces are redundant.

Reply:
now uses .trim()